### ISSUE TITLE
'atomac.__a11y.Error' on running LDTP with ldtp.getwindowlist()

### ISSUE THREAD

**Issue by muffadaldaginawala** [pyatom/pyatom#145]
I am trying to run LDTP on my mac OS X machine. I have enabled accessibility and disabled SIP . 

I get the following error when running getwindowlist(): 

Fault: <Fault 1: "<class 'atomac.a_a11y.Error'>:Error retrieiving attribute (AXError -25201)">

I am running OS X on a macmini

Does anyone know what is going on?

Thanks,
Muffadal

**Comment by muffadaldaginawala** [comment]
So after Looking around for quite a while. 

I uninstalled LDTP( the production version(atomac 1.1.0)) and installed the dev version(1.2.0) from here https://github.com/pyatom/pyatom/archive/master.zip (sudo your_python setup.py build; sudo your_python setup.py develop) . 

It seems to have solved the problem and everythng works smoothly now. 

Seems to me as if production version should be fixed to resolve this issue since i think the majority will do a "pip install" and be getting 1.1.0 and might bump into this issue.

**Comment by suresh1227** [comment]
Hi, muffadaldaginawala 

I have installed new version of ldtp any configuration settings we need to do in the bash profile please let me know. e.g.:environmental variables 

please help me on this.

**Comment by muffadaldaginawala** [comment]
No, you just need to "pip install". If you encounter the same issue i did
after that then install the dev version as i have mentioned in the comment.

**Comment by suresh1227** [comment]
Thanks a lot muffadaldaginawala

its working now .

**Comment by suresh1227** [comment]
Hi muffadaldaginawala,

i need a help.

i want to automate mac login panel, i got title of that but i am not able
to read or click on that. since when i call activate method window not
becomeing active.

*please help me on this. i added in accessibility also.*

Regards
Suresh.R

**Comment by suresh1227** [comment]
I want to automate login process please kindly help me.

bugndleid i given and got the titile.

i am not able to get other things.

Please help me

regards
Suresh.R

**Comment by suresh1227** [comment]
please . help me on this

**Comment by suresh1227** [comment]
Hi,

Kindly let me know how to get list objects, i have one list box i want
elements in that list box.

Please help me on this. i  am using atomac in mac.

Regards
Suresh.R

**Comment by muffadaldaginawala** [comment]
https://ldtp.freedesktop.org/user-doc/d1/d7c/a00003.html

Check out ComboBox from here. Maybe that will help .

Thanks,
Muffadal

**Comment by suresh1227** [comment]
Hi, muffadaldaginawala,

can you please tell me how to read static or any test in application. i am
having atomac.AXclasses.Nativeelement AXwindow and AXGroup, AXStaticText.
i have used object.staticTexts() and all the functions but i am not able to
read. kinldy help me
i am doing automation on MacOS

Regards
Suresh.R

**Comment by sagarks** [comment]
I tried building with https://github.com/pyatom/pyatom/archive/master.zip 
lunchapp fails on OS X 10.12
Debug info does not show any error.
